modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
Hi, Alex, thank you, this helps. Need your advice though: $ENV{OPENCV_TEST_DATA_PATH} - is this cmake variable or environment? I did not find it in CMakeVars.txt. The closest match was OPENCV_TEST_DATA_INSTALL_PATH=share/OpenCV/testdata. As far as I can see it is used currently only by text module.
The other question - do I have to copy space_shuttle to one of the build/ subfolders during cmake? Which one I need to use?